spec/models/item_spec.rb

```diff
@@ -6,4 +6,16 @@
     it { should have_many :invoice_items }
     it { should have_many :invoices }
   end
+
+  context 'business analytics' do
+  	describe '#best_day' do
+  		it 'returns the best day' do
+  			item 		= create(:item)
+  			invoice = create(:invoice)
```

Comment:
@Dpalazzari can you add another invoice with a different day to verify that it grabs the right day?